src/api/audits.coffee

```diff
@@ -0,0 +1,129 @@
+Audit = require('../model/audits').Audit
+authorisation = require './authorisation'
+Q = require 'q'
+logger = require 'winston'
+utils = require "../utils"
+
+
+# function to construct projection object
+getProjectionObject = (filterRepresentation) ->
+  switch filterRepresentation
+    when "simpledetails"
+      # view minimum required data for audit details view
+      return {}
+    when "full"
+      # view all audit data
+      return {}
+    else
+      # no filterRepresentation supplied - simple view
+      # view minimum required data for audits
+      return { "participantObjectIdentification": 0, "activeParticipant": 0, "rawMessage": 0 }
+  
+
+
+###
+# Adds a Audit
+###
+exports.addAudit = ->
```

Comment:
Just to open this up to discussion - do you we want to support custom add endpoints for auditing (an addition to the standards-based ones)? I personally think it's okay to leave this in - we could view it as a "value add" service that the HIM is providing...

@rcrichton what do you think?